app/helpers/token.js

```diff
@@ -0,0 +1,15 @@
+const jwt = require('jsonwebtoken');
+
+const secret = 'shhhhh';
```

Comment:
I will let you pass this but move this `secret ` to a constant in config/constants